### DISCUSSION TITLE
Sharpness option in Simplemenu?

### DISCUSSION THREAD

**Discussion by ryan0x7d6** [eduardofilo/RG350_adam_image#109] [General]
Device - RG280V
Adam Image version - v1.3

Hello, amazing work on the Adam Image, I think having a unified firmware approach for multiple devices is such a great idea!
I was poking around the Simplemenu settings and noticed an option, when pressing start, then going to system, there is an option labelled "Sharpness" that seems to do nothing. I'm wondering if it actually doesn't do anything, or if I'm just not noticing it? I remember on my RG350, on the stock firmware (not OpenDingux Beta), pushing POWER + DPAD-DOWN would turn down the sharpness in certain emulators, making things like GBA games look so much better. Is this possible with the Adam image?

**Comment by eduardofilo** [comment]
Hi.
I think that sharpness option applies when emulators (standalone ones, not RetroArch that handles things on its own) do hardware scaling. I have tried with FBA and I have noticed quite a difference.
As for the keyboard shortcuts that were in stock, it is something that must be entered at the system level.
